ws-nextjs-app/pages/[service]/send/[id]/GenericMessage/index.tsx

```diff
@@ -10,17 +10,13 @@ interface Props {
 }
 
 const GenericMessage = ({ heading, children }: Props) => {
+  const el = useRef<HTMLHeadingElement>(null);
+  useEffect(() => el.current?.focus(), []);
   return (
     <>
-      <Heading
-        level={1}
-        id="content"
-        tabIndex={-1}
-        css={styles.heading}
-        size="trafalgar"
-      >
+      <h1 id="content" ref={el} tabIndex={-1} css={styles.heading}>
```

Comment:
Are we using an `h1` here to make use of the `ref` prop?

Reply:
Yep. I wasn't able to add ref attribute to the Heading component so I used a H1. I believe this might be possible if I add forwardRef to the Heading component ([as per this](https://react.dev/learn/manipulating-the-dom-with-refs#accessing-another-components-dom-nodes)) - but not sure if this is a good idea?

Reply:
It was an error along the lines of '`Property 'ref' does not exist on type 'IntrinsicAttributes & { css?: Interpolation<Theme>; } & Props'.ts`'- so there maybe a way to override this.

Reply:
Tbh I've not used forwardRef much, but it may be worth a try to see if it works? Would be good to re-use the component, but not a big deal if we can't in this case.

Reply:
We're taking a look! It does mean updating Heading and Text to take forwardRef.